system/filesystem/check-checksums.yml

```diff
@@ -0,0 +1,105 @@
+---
+type: CheckConfig
+api_version: core/v2
+metadata:
+  name: check-checksums
+  namespace: default
+spec:
+  command: check-checksums.rb -f /bin/sh -h eca410732dd8568ec169c1879965f4361bd69d72c06cb5990c7b3962f197a844
```

Comment:
What does -h stand for here? Could we replace this value with a label?

Reply:
It's the sha256 checksum of `/bin/sh` and both values should probably be replaced with labels.